torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -494,6 +494,8 @@ struct Environment {
           {"ord", std::make_shared<BuiltinFunction>(aten::ord, at::nullopt)},
           {"chr", std::make_shared<BuiltinFunction>(aten::chr, at::nullopt)},
           {"bin", std::make_shared<BuiltinFunction>(aten::bin, at::nullopt)},
+          {"AssertionError",
```

Comment:
I think we should only bind `Exception` which is the base class that has 62 other classes that inherit from it.

Reply:
`AssertionError` is special here since we don't have real assert statements, we don't really need to add any others. This PR changes them into, so the compiler needs to know about `AssertionError`:
```python
if not cond:
    raise AssertionError(message)
```

Reply:
What do you think ?

Reply:
Added a comment, why should we bind in `Exception`? If its for things like AD formulas I think we can just wait and see what is specifically requested before adding it